components/centraldashboard/build_image.sh

```diff
@@ -3,21 +3,24 @@
 # A simple script to build the Docker images.
 # This is intended to be invoked as a step in Argo to build the docker image.
 #
-# build_image.sh ${DOCKERFILE} ${IMAGE} ${TAG} (optional | ${TEST_REGISTRY})
+# build_image.sh ${DOCKERFILE} ${IMAGE} ${TAG}
 set -ex
 
 DOCKERFILE=$1
 IMAGE=$2
 TAG=$3
+CONTEXT_DIR=$(dirname "$DOCKERFILE")
+PROJECT="{GCP_PROJECT}"
```

Comment:
Is there a reason you are passing these as environment variables and the others as command line arguments?

Reply:
Actually looking at the workflow I think it makes sense to continue to pass as environment variables since they're optional; e.g. only needed in case of use of GCB.

Reply:
👍